esp-alloc/src/lib.rs

```diff
@@ -280,6 +556,15 @@ unsafe impl GlobalAlloc for EspHeap {
                     region.heap.deallocate(NonNull::new_unchecked(ptr), layout);
                 }
             }
+
+            #[cfg(feature = "internal-heap-stats")]
+            {
+                let mut internal_heap_stats = self.internal_heap_stats.borrow_ref_mut(cs);
+                drop(regions);
+                // We need to call used because [linked_list_allocator::Heap] does internal size
```

Comment:
I would write it that way probably, but again, feel free to ignore!
```suggestion
                // We need to call `used()` because [linked_list_allocator::Heap] does internal size
```